### PR TITLE
profile & expose: fix integration with contacts

### PR DESCRIPTION
**Pull request by mikolajpp**
There were some outdated type references in %expose that were preventing the upgrade. Additionally, both %profile and %expose were using the removed `/contact` API endpoint. For now we just migrate to subscribing to `/news` and filtering on our own profile updates. We should migrate these two agents to use new %contacts API soon.

### CODE REVIEW

**Comment by mikolajpp** [comment]
Fixes TLON-3253

File: desk/app/expose.hoon [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ::    then visit in the browser:
 ::    /expose/that/reference/as/copied/123456789
 ::
-/-  c=cite, d=channels, co=contacts
+/-  c=cite, d=channels, t=contacts-0
```
Comment (Fang-) [inline desk/app/expose.hoon:8]: Can we keep the old import names, here and in other files, please?

**Review by Fang- — COMMENTED** [review]
The import name change here seems unwarranted/out of scope.

Also, don't both of these agents need `+on-load` logic, to tear down the old subscription and set up the new one?

File: desk/app/expose.hoon (hunk 1 above)
Comment (mikolajpp) [reply desk/app/expose.hoon:8]: We can – although we avoid the name clash with `+co` arm in this way.

File: desk/app/expose.hoon [hunk 2]
```diff
@@ -254,10 +279,10 @@
       ==
     ==
   ::
-      [%contacts ~]
+      [%contacts %news ~]
     ?-  -.sign
       %poke-ack  !!
-      %kick      [[%pass /contacts %agent [our.bowl %conacts] %watch /contact]~ this]
```
Comment (Fang-) [inline desk/app/expose.hoon:260]: lol, oops, good catch

**Review by Fang- — APPROVED** [review]
Looks good, thank you!

**Pull request merged**